Sources/DependenciesMacrosPlugin/DependencyEndpointMacro.swift

```diff
@@ -162,14 +162,14 @@ public enum DependencyEndpointMacro: AccessorMacro, PeerMacro {
       }
       let appliedParameters =
         parameters
+        .enumerated()
         .map {
-          guard let typed = $0.type.as(AttributedTypeSyntax.self),
-            typed.specifier?.tokenKind == .keyword(.inout)
-          else { return false }
-          return true
+          $1.isInout
+          ? "&p\($0)"
+          : $1.isAutoclosure
+          ? "p\($0)()"
+          : "p\($0)"
```

Comment:
#159 style seems so cool that I had to try it myself. Originally, I was writing code like this 😅

```swift
let appliedParameters =
  parameters
  .enumerated()
  .map {
    if let typed = $1.type.as(AttributedTypeSyntax.self),
       typed.specifier?.tokenKind == .keyword(.inout) {
      return "&p\($0)"
    }
    
    if let typed = $1.type.as(AttributedTypeSyntax.self),
       typed.attributes.contains(where: {
         $0.as(AttributeSyntax.self)?.attributeName.as(IdentifierTypeSyntax.self)?.name.tokenKind == .identifier("autoclosure")
       }) {
      return "p\($0)()"
    }

    return "p\($0)"
  }
  .joined(separator: ", ")
```